### PR TITLE
Education use case addition

### PR DESCRIPTION
**Pull request by cviddenKwantum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,12 @@ With these valuable insights, the customer service representative can make concr
 
 - **Health Care** - Medical professionals are working to ensure there is enough hospital capacity Covid patients. They use a variety of data about infected patients to identify those likely to require hospitalization. The dashboard can also guide their treatments, to help reduce the number of hospitalizations. 
 
+- **Education** - Student attrition rates are a key performance indicator for any higher education provider and have a significant
+impact on students. Understanding what causes student attrition (ie financial status, academic performance, or student wellbeing)
+is a complex challenge which is often specific to individual student circumstances. The RAI dashboard can help higher education decision makers better understand student attrition while informingthe creation of intervents to improve student support services and foster student engagment and belonging.
```
Comment (imatiach-msft) [inline README.md:67]: "while informingthe creation of intervents to improve student support services"
can this be rephrased?  the text looks confusing.

File: README.md [hunk 2]
```diff
@@ -122,19 +128,31 @@ if you need additional guidance with this task.
...
+    refer to this [Health Care scenario guide](./documentation/educationExample.md) 
     for suggestions on how to interpret and apply the Responsible AI dashboard.  
     <br />
 
+3. **Education**
+    In addition to several components that can help you improve model performance, 
+    this scenario highlights a causal analysis tool that can guide real-world action. 
+    This notebook takes about **25-30 minutes** to create the dashboard.  
+    
+    - Follow this link to the [Jupyter notebook](./healthcare_story/Covid_Healthcare_Dashboard.ipynb).
```
Comment (imatiach-msft) [inline README.md:149]: it looks like you mixed up the links to covid and education here, can you please fix here and above?

File: education_story/Education_Dashboard.ipynb [hunk 3]
```diff
@@ -0,0 +1,1986 @@
...
...00879     0.599913   \n9997     0.210662        0.000000               0.017499     0.720440   \n9998     0.080218        0.042558               0.005508     0.774354   \n9999     0.026392        0.001591               0.021704     0.861997   \n\n       Adjunct   Faculty  Unknown_IntructorType  PELL_Eligible  Attrition  \n0     0.707361  0.120330               0.189601              0  Attrition  \n1     0.347302  0.532975               0.098299              0  Attrition  \n2     0.311391  0.730372               0.000000              0     Retain  \n3     0.458737  0.331574               0.244544              0  Attrition  \n4     0.306996  0.727143               0.000000              0     Retain  \n...        ...       ...                    ...            ...        ...  \n9995  0.434124  0.432218               0.146807              0     Retain  \n9996  0.298951  0.495972               0.194673              0  Attrition  \n9997  0.456662  0.422180               0.137719              0  Attrition  \n9998  0.336495  0.399804               0.253111              0  Attrition  \n9999  0.127220  0.759477               0.109919              0     Retain  \n\n[10000 rows x 33 columns]",
```
Comment (imatiach-msft) [inline education_story/Education_Dashboard.ipynb:363]: please remove notebook outputs

**Comment by imatiach-msft** [comment]
can you download csv file instead of putting it on github?  we can put it in our blob storage account.  maybe we can setup a quick session next week to do this together.

File: README.md (hunk 1 above)
Comment (cviddenKwantum) [reply README.md:67]: Yes, will rephrase.

**Review by imatiach-msft — APPROVED** [review]

**Pull request merged**